src/assets/node.sh

```diff
@@ -1,17 +1,47 @@
 #!/bin/bash
 NODE_VERSION=<%= nodeVersion %>
 NPM_VERSION=<%= npmVersion %>
+MAJOR_NODE_VERSION=`echo $NODE_VERSION | awk -F. '{print $1}'`
+MINOR_NODE_VERSION=`echo $NODE_VERSION | awk -F. '{print $2}'`
+METEOR_VERSION=<%= meteorVersion %>
 
+echo "Node: $NODE_VERSION"
+echo "Major: $MAJOR_NODE_VERSION"
+echo "Minor: $MINOR_NODE_VERSION"
 
-export NVM_DIR="/.nvm"
-# Install nvm
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.6/install.sh | bash
-[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"
+if [[ $MAJOR_NODE_VERSION == "14" && $MINOR_NODE_VERSION -ge 21 ]]; then
```

Comment:
It might be helpful to look at how we fixed this in the zodern:meteor docker image: https://github.com/zodern/meteor-docker/blob/467ec350e3a4d6ba93987b8d14eaa8cc79959a67/image/scripts/setup_nvm.sh#L21-L38

It copies Meteor's node into nvm's folder so nvm is able to use it. You'll also want to check the patch version, since Meteor's node version uses the same minor version as some official releases.